cpp/src/arrow/compute/kernels/scalar_arithmetic.cc

```diff
@@ -1495,6 +1516,21 @@ const FunctionDoc log1p_checked_doc{
      "Use function \"log1p\" if you want non-positive values to return "
      "-inf or NaN."),
     {"x"}};
+
+const FunctionDoc floor_doc{
+    "Calculate the greatest integer less than or equal to the argument element-wise",
+    "",
+    {"x"}};
+
+const FunctionDoc ceiling_doc{
+    "Calculate the least integer greater than or equal to the argument element-wise",
+    "",
+    {"x"}};
+
+const FunctionDoc truncate_doc{
+    "Calculate the nearest integer not greater than to the argument element-wise",
```

Comment:
This means we truncate towards negative infinity? (e.g. truncate(-1.1) = -2 since -1 > -1.1?)

Reply:
Ah, not greater in *magnitude* according to cppreference. Maybe that should be clarified.